app/assets/javascripts/active_admin_chat/vendor.js

```diff
@@ -0,0 +1,1077 @@
+'use strict';
```

Comment:
Where did you get this from? cable_ready?

Reply:
I transpiled the cable ready JS manually in order to use it in the asset pipeline, and the transpiler I used put all the code you see in this file (except some minor modifications I made to make it work)